opennms-doc/guide-admin/src/asciidoc/text/service-assurance/monitors/SSLCertMonitor.adoc

```diff
@@ -54,13 +62,14 @@ This monitor implements the <<ga-service-assurance-monitors-common-parameters, C
 | `${svcname}`    | The service name
 |===
 
-WARNING: The monitor has no support for communicating on other protocol layers above the SSL session layer. It
-is not able to send a Host header for HTTPS, or issue a STARTTLS command for IMAP, POP3, SMTP, FTP, XMPP, LDAP,
+WARNING: The monitor has limited support for communicating on other protocol layers above the SSL session layer. The
+STARTTLS support has only been tested with a single XMPP server. It is not known if the same approach will prove
```

Comment:
Can we apply the one-sentence-per line rule in this file as described in the asciidoc-recommended-practices.